src/platform/renesas/DiagnosticDataProviderImpl.cpp

```diff
@@ -93,6 +93,42 @@ CHIP_ERROR DiagnosticDataProviderImpl::GetBootReason(BootReasonType & bootReason
     return err;
 }
 
+CHIP_ERROR DiagnosticDataProviderImpl::GetNetworkInterfaces(NetworkInterface ** netifpp)
+{
+    struct netif * net_interface;
+    CHIP_ERROR err = CHIP_NO_ERROR;
+    NetworkInterface * ifp = new NetworkInterface();
+
+    net_interface = netif_find("en0");
```

Comment:
we should remember to add a task to the wifi story to extend this with both/all.

Reply:
Done: https://sensorfy.atlassian.net/browse/TSBM-236

Reply:
Didn't you find the classes to iterate over all lwip interfaces?

Reply:
Yes, I've updated code with Inet::InterfaceIterator